Makefile

```diff
@@ -70,19 +70,18 @@ benchmarks/refmt_main3b.cmx: benchmarks/refmt_main3b.ml
 lib/test.exe: $(TEST_FILES)
 	$(OCAMLOPT) $(OCAMLFLAGS) -O2 -o ./lib/test.exe -bin-annot -I +compiler-libs ocamlcommon.cmxa -I src -I tests $(TEST_FILES)
 
-test: build-native lib/test.exe
-	./node_modules/.bin/reanalyze -all-cmt . -suppress tests -suppress benchmarks
-	./test.sh
+test: reanalyze build-native lib/test.exe
 	./lib/test.exe
+	./test.sh
 
-roundtrip-test: bootstrap lib/test.exe
-	./node_modules/.bin/reanalyze -all-cmt . -suppress tests -suppress benchmarks
-	ROUNDTRIP_TEST=1 ./test.sh
+roundtrip-test: reanalyze bootstrap lib/test.exe
```

Comment:
does the `-suppress benchmarks` still matter?

Reply:
Removed